lib/Sema/CSGen.cpp

```diff
@@ -3884,7 +3885,7 @@ bool ConstraintSystem::generateConstraints(
                                                getConstraintLocator(typeRepr));
     setType(typeRepr, backingType);
 
-    auto propertyType = wrappedVar->getType();
+    auto propertyType = getVarType(wrappedVar);
```

Comment:
This code path is only taken for declared property wrappers without an initial value, e.g.

```
@Wrapper var value: Int
```

I think if we were to replace an error type with a hole type for something like `@Wrapped var value: Undeclared`, no fixes would be applied. Does that matter?

Reply:
Fixes don't really matter for code completion at the moment, maybe in the future we'd need to create a separate fix to denote that solver is using an invalid declaration but I can't imagine where it would be useful...